kong/db/schema/typedefs.lua

```diff
@@ -331,6 +331,14 @@ typedefs.url = Schema.define {
   description = "A string representing a URL, such as https://example.com/path/to/resource?q=search."
 }
 
+
+typedefs.cookie_name = Schema.define {
+  type = "string",
+  custom_validator = utils.validate_cookie_name,
```

Comment:
If the function is only used here, please remove it from utils and into typedefs.lua

Reply:
It's unit-tested so it needs to be exported. Also before the patch, this function was already exported and in the util module.